synapseAnnotations/data/cancer.json

```diff
@@ -247,6 +252,11 @@
         "description": "Acute lymphoblastic leukemia of T-cell origin. It comprises about 15% of childhood cases and 25% of adult cases. ",
         "source":"http://purl.obolibrary.org/obo/NCIT_C3183"
       },
+      {
+        "value": "Chronic Lymphocytic Leukemia",
```

Comment:
Could you match the capitalization from http://purl.obolibrary.org/obo/DOID_1040?